ddtrace/_monkey.py

```diff
@@ -91,6 +91,7 @@
     "tornado": False,
     "openai": True,
     "langchain": True,
+    "anthropic": True,
```

Comment:
This integration is marked as beta but it will be auto enabled. Can we make this integration opt in?